src/App.tsx

```diff
@@ -7,14 +7,109 @@ import Header from "./components/Header";
 import Home from "./components/Home";
 import Stage from "./components/Stages";
 import AddStage from "./components/Stages/AddStage";
+import { BsMoonFill } from "react-icons/bs";
+import { BiSun } from "react-icons/bi";
 
 function App() {
   const style = {
-    wrapper: "w-full flex flex-row justify-center max-w-screen-2xl px-[200px]",
+    wrapper:
+      "w-full flex flex-row justify-center max-w-screen-2xl px-[200px] dark:bg-dark-background min-h-screen items-start",
     container: "flex flex-col justify-center mt-6",
+    themeIcon: "text-xl m-2",
+    themeIconMoonLight: "text-xl m-2 text-light-font-lightV2",
+    themeContainer:
+      "fixed bottom-2 right-2 flex justify-between items-center bg-light-primary-grey dark:bg-white py-1 px-2 rounded-[30px] z-50",
+    iconDiv: "cursor-pointer",
+  };
+
+  const [isDark, setIsDark] = useState(false);
```

Comment:
This variable is not needed, can check the html class if it's dark or not.

Reply:
Done